hoomd/tune/custom_tuner.py

```diff
@@ -74,3 +75,11 @@ class _InternalCustomTuner(_InternalCustomOperation, Tuner):
 
     def tune(self, timestep):
         return self._action.act(timestep)
+        """
+        .. deprecated:: 5.0.0
```

Comment:
```suggestion
        .. deprecated:: 4.5.0
```
We document the version it was first deprecated in, not the version in which it will be removed.